crates/subspace-service/src/sync_from_dsn/snap_sync.rs

```diff
@@ -72,18 +94,22 @@ pub(crate) async fn snap_sync<Block, AS, Client, PG>(
             import_queue_service.as_mut(),
             &sync_service,
             &network_service_handle,
-            None,
+            target_block,
             &erasure_coding,
         );
 
         match snap_sync_fut.await {
-            Ok(()) => {
+            Ok(success) => {
                 debug!("Snap sync finished successfully");
+
+                success
             }
             Err(error) => {
                 error!(%error, "Snap sync failed");
+
+                false
             }
-        }
+        };
```

Comment:
The return value of this future is `Result<bool>` but it seems like `Result<()>` would be enough as the `bool` is not used and dropped right away.

Reply:
This code has already been changed in the main branch. I'll merge it after the last approval.